wordpress/wp-content/plugins/cds-base/classes/Modules/ListManager/app/src/components/ListViewTable.tsx

```diff
@@ -177,13 +176,13 @@ export const ListViewTable = () => {
                         Header: 'Upload',
                         accessor: 'active',
                         Cell: ({ row }: { row: any }) => {
-                            return <UploadListLink id={`${row?.values?.id}`} />
+                            return <UploadListLink serviceId={serviceId} listId={`${row?.values?.id}`} />
```

Comment:
These are returning "same text for different links" error. Easiest fix is to do "Upload list (to Paul's awesome subscribers)".